data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -92,26 +93,27 @@ def transform(self) -> None:
 
         logger.info("Pruning Census GeoJSON")
         fields = [self.GEOID_FIELD_NAME, self.GEOMETRY_FIELD_NAME]
-        self.geojson_usa_df = self.geojson_usa_df[fields]
 
         logger.info("Merging and compressing score CSV with USA GeoJSON")
-        self.geojson_score_usa_high = self.score_usa_df.merge(
-            self.geojson_usa_df, on=self.GEOID_FIELD_NAME, how="left"
+        self.geojson_score_usa_high = self.score_usa_df.set_index(
```

Comment:
I changed this slightly so it would run ever so slightly faster

Reply:
Looks good!